packages/design-tokens/src/scss/themes/theme-light/_colors.scss

```
@@ -0,0 +1,421 @@
+            default: $action-link-tertiary-default,
+            hover: $action-link-tertiary-hover,
+        ),
+        visited: (
+            default: $action-link-visited-default,
+        ),
+    ),
+    toggle: (
+        border-basic: $action-toggle-border-basic,
+        border-subtle: $action-toggle-border-subtle,
+        content-basic: $action-toggle-content-basic,
+        content-subtle: $action-toggle-content-subtle,
+        default-basic: $action-toggle-default-basic,
+        default-subtle: $action-toggle-default-subtle,
+        hover-basic: $action-toggle-hover-basic,
+        hover-subtle: $action-toggle-hover-subtle,
+    ),
+) !default;
+
+$background-colors: (
+    backdrop: $background-backdrop,
+    brand: (
+        primary: $background-brand-primary,
+        secondary: $background-brand-secondary,
+    ),
+    form-field: (
+        active: $background-form-field-active,
+        default: $background-form-field-default,
+        hover: $background-form-field-hover,
+    ),
+    interactive: (
+        active: $background-interactive-active,
+        default: $background-interactive-default,
+        hover: $background-interactive-hover,
+        selected: $background-interactive-selected,
+    ),
+    primary: $background-primary,
+    secondary: $background-secondary,
+    tertiary: $background-tertiary,
+) !default;
+
+$border-colors: (
+    interactive: (
+        active: $border-interactive-active,
+        danger: $border-interactive-danger,
+        default: $border-interactive-default,
+        hover: $border-interactive-hover,
+        informative: $border-interactive-informative,
+        selected: $border-interactive-selected,
+        success: $border-interactive-success,
+        warning: $border-interactive-warning,
+    ),
+    secondary: (
+        default: $border-secondary-default,
+    ),
+    focus: (
+        default: $border-focus-default,
+    ),
+) !default;
+
+$custom-colors: (
+    01: (
+        active: $custom-01-active,
+        background-basic: $custom-01-background-basic,
+        background-subtle: $custom-01-background-subtle,
+        border-basic: $custom-01-border-basic,
+        border-subtle: $custom-01-border-subtle,
+        content-basic: $custom-01-content-basic,
+        content-subtle: $custom-01-content-subtle,
+        default: $custom-01-default,
+        hover: $custom-01-hover,
+    ),
+    02: (
+        active: $custom-02-active,
+        background-basic: $custom-02-background-basic,
+        background-subtle: $custom-02-background-subtle,
+        border-basic: $custom-02-border-basic,
+        border-subtle: $custom-02-border-subtle,
+        content-basic: $custom-02-content-basic,
+        content-subtle: $custom-02-content-subtle,
+        default: $custom-02-default,
+        hover: $custom-02-hover,
+    ),
+) !default;
+
+$disabled-colors: (
+    background: $disabled-background,
+    border: $disabled-border,
+    foreground: $disabled-foreground,
```

Shouldn't it be `content` so it matches similar token sets?

```suggestion
    content: $disabled-content,
```

---

Good point, I opened a discussion with our designers and we can solve this one in another PR.